tfjs-backend-webgpu/src/kernels/binary_op_util.ts

```diff
@@ -208,14 +208,14 @@ const POW_WGSL = `
   if (b == 0.0) {
     return 1.0;
   }
-  if (i32(round(mod(b, 2.0))) != 1) {
+  if (round(abs(b)) % 2.0 != 1.0) {
```

Comment:
Should be `if (round(abs(b) % 2.0) != 1)` if we strictly map to the old code?

Reply:
Done